package/kedro_viz/launchers/cli/lazy_default_group.py

```diff
@@ -0,0 +1,76 @@
+"""`kedro_viz.launchers.cli.lazy_default_group` provides a custom mutli-command
```

Comment:
You can just move this class to the `utils.py` if you want like in https://github.com/kedro-org/kedro/pull/3883

Reply:
I saw that on kedro. I felt we have utility functions in utils and have the lazy class as a separate file. Thank you